README.md

```diff
@@ -0,0 +1,39 @@
+# Icinga DB
+
+1. [About](#about)
+2. [Documentation](#documentation)
+3. [Installation](#installation)
+4. [Configuration](#configuration)
+
+## <a id="About"></a>
+
+The Icinga DB web module is the way to go, to interact with the new database and to use the benefits it brings.
+
+This module reimplements and aims to replace the old monitoring module.
+It keeps all functionality from its predecessor and adds new quality of life features, such as:
+- bulk operations: set entire host groups into downtime with one operation
+- modals instead of tabs to speed up the workflow
+- a view switcher, that enables the user to choose how much detail should be displayed
+- the status bar now checks for set filters
```

Comment:
I would replace the above paragraph entirely and avoid comparisons with the old module where applicable. It should be catchy and encouraging to try things immediately. The "About" section is where users read and continue, or close the page.


```
The Icinga DB web module pushes your monitoring stack to the next level. 

Based on the lessons learnt with the base monitoring module, it offers a modern and streamlined design to provide a clear and concise overview on your monitoring environment.

The module adds new features, such as:

- Bulk operations to schedule downtimes for host groups and alike.
- Enhanced action workflows with In-window modals 
- View switching to chose how much detail will be displayed
- Improved status bar with applied filters
```

Reply:
Changed, thanks for the suggestion!